### PR TITLE
my homework for week0

### PR DESCRIPTION
**Pull request by jawkha**
I created the Person constructor using the ES6 class method in Person.js and using the ES5 constructor function in PersonES5.js.

In index.js, line 2 has been commmented out which imports PersonES5.js.

### CODE REVIEW

File: week0/homework/PersonES5.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	this.call = function() {
+		if (this.phone) {
+			console.log('calling ' + this.name + ' at ' + this.phone + '...');
+		} else {
+			console.log(this.name + ' has no phone number saved.');
+		}
+	};
+
+	this.birthday = function() {
+		console.log(
```
Comment (r0hitsharma) [inline week0/homework/PersonES5.js]: @jawkha , here's the only thing i was expecting a bit differently, was to increment the age, and not just display the incremented age, but of course it wasn't specifically mentioned in the instruction.

Comment (r0hitsharma) [inline week0/homework/PersonES5.js]: @jawkha, here's the only thing i was expecting a bit differently, was to increment the age, and not just display the incremented age, but of course it wasn't specifically mentioned in the instruction.

**Review by r0hitsharma — APPROVED** [review]
Works well overall :)

**Comment by jawkha** [comment]
I made changes to the birthday function. I think it should be ok now.

**Review by nynnejc — APPROVED** [review]

**Pull request merged**